src/hotspot/cpu/riscv/stubRoutines_riscv.cpp

```diff
@@ -479,5 +479,17 @@ ATTRIBUTE_ALIGNED(4096) juint StubRoutines::riscv::_crc_table[] =
     0x29413c29, 0x548c7116, 0xd2dba657, 0xaf16eb68, 0x05050e94,
     0x78c843ab, 0xfe9f94ea, 0x8352d9d5, 0x71c95953, 0x0c04146c,
     0x8a53c32d, 0xf79e8e12, 0x5d8d6bee, 0x204026d1, 0xa617f190,
-    0xdbdabcaf
+    0xdbdabcaf,
+
+    // CRC32 table for carry-less multiplication implementation
+    0xe88ef372UL, 0x00000001UL,
+    0x4a7fe880UL, 0x00000001UL,
+    0x54442bd4UL, 0x00000001UL,
+    0xc6e41596UL, 0x00000001UL,
+    0x3db1ecdcUL, 0x00000000UL,
+    0x74359406UL, 0x00000001UL,
+    0xf1da05aaUL, 0x00000000UL,
+    0x5a546366UL, 0x00000001UL,
+    0x751997d0UL, 0x00000001UL,
+    0xccaa009eUL, 0x00000000UL,
```

Comment:
Did you calculate these yourself, or where do they come from?

Reply:
They're from aarch64 implementation, and I think x86 use the same ones.